packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -321,28 +346,30 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
             ...unhandledProps,
           })}
         >
-          {createShorthand(Input, input, {
-            defaultProps: () =>
-              getA11yProps('input', {
-                placeholder: props.inputPlaceholder,
-                disabled: props.disabled,
-                error: !!error,
-                value: formattedDate,
-                readOnly: !allowManualInput,
-                required: props.required,
-                ref: inputRef,
-                'aria-label': formatRestrictedInput(restrictedDatesOptions, dateFormatting),
-              }),
-            overrideProps: overrideInputProps,
-          })}
+          {type !== DatepickerType.ButtonOnly &&
+            createShorthand(Input, input, {
+              defaultProps: () =>
+                getA11yProps('input', {
+                  placeholder: props.inputPlaceholder,
+                  disabled: props.disabled,
+                  error: !!error,
+                  value: formattedDate,
+                  readOnly: !allowManualInput,
+                  required: props.required,
+                  ref: inputRef,
+                  'aria-label': formatRestrictedInput(restrictedDatesOptions, dateFormatting),
+                }),
+              overrideProps: overrideInputProps,
+            })}
           {createShorthand(Popup, popup, {
             defaultProps: () => ({
               open: openState && !props.disabled,
               content: calendarElement,
               trapFocus: {
                 disableFirstFocus: true,
               },
-              trigger: <Button icon={<CalendarIcon />} title="Open calendar" iconOnly disabled={props.disabled} />,
+              trigger: triggerButtonElement,
+              target: type === DatepickerType.InputOnly ? inputRef.current : null,
```

Comment:
Please let's make `input` to be default `target` for `InputOnly` and `InputAndButton` cases and have the following default positioning:
In all cases - `position: 'below'` and `align:'start'` default props for `Popup`

Reply:
Fixed.